api/src/auth/auth.module.ts

```diff
@@ -17,7 +17,7 @@ import { TfaModule } from 'src/tfa/tfa.module';
         UsersModule, PassportModule, forwardRef(() => TfaModule),
         JwtModule.register({
             secret: process.env.JWT_SECRET,
-            signOptions: { expiresIn: '2h' },
+            signOptions: { expiresIn: '30d' },
```

Comment:
It is possible to create a variable that dynamically adds the last day of a month to the expiry date, as not all months end on the 30th day. February ends on the 28th in leap year.